samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -40,14 +40,33 @@ tests:
     integration_platforms:
       - nrf9160dk_nrf9160_ns
     platform_allow: nrf9160dk_nrf9160_ns
-    extra_args: SHIELD=nrf7002_ek OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    extra_args: SHIELD=nrf7002ek_nrf7002 OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
```

Comment:
Commenting here to get a conversation but you'll need a change of `SHIELD` variable for `applications/asset_tracker_v2/sample.yaml` also to get CI through.

Reply:
Done

> In general, instead of having multiple copies of the same file, please try to have a single file called nrf700x_* if the contents are the same for all variants (or even if a single file is possible using conditional statements for different variants)

Having a file of nrf700x format is possible for only dts bindings. But setting a generic "compatible" string in dts bindings is interfering with the checks we have added for excluding supplicant support for nrf7000 device. So, not making this change for now.

Reply:
Done